docs/en/revocation-lists.rst

```diff
@@ -212,29 +227,59 @@ The following HTTP Status Codes MUST be supported:
     * - **Status Code**
       - **Body**
       - **Description**
-    * - *204 No Content*
-      - 
-      - The Digital Credential has been successfully revoked.
+    * - *200 Created*
+      - Revocation Assertion Response
+      - The Revocation Assertion Response has been successfully created.
     * - *400 Bad Request*
       - Error code and description
-      - The issuer cannot fulfill the request because of invalid parameters.
-    * - *404 Not Found*
-      - 
-      - The Digital Credential to be revoked can not be found by the Issuer.
+      - The Credential Issuer cannot fulfill the request because of invalid parameters.
     * - *500 Internal Server Error*
       - 
-      - The Issuer encountered an internal problem. (:rfc:`6749#section-5.2`).
+      - The Credential Issuer encountered an internal problem. (:rfc:`6749#section-5.2`).
     * - *503 Service Unavailable*
       - 
-      - The Issuer is temporary unavailable. (:rfc:`6749#section-5.2`).
+      - The Credential Issuer is temporary unavailable. (:rfc:`6749#section-5.2`).
+
+The ``revocation_assertion_responses`` object MUST contain the following mandatory claims.
+
+.. _table_http_response_claim:
+.. list-table:: 
+    :widths: 20 60 20
+    :header-rows: 1
 
-For HTTP error responses that involve a body, the body MUST be encoded in ``application/json`` format and MUST contain the following parameters:
+    * - **Claim**
+      - **Description**
+      - **Reference**
+    * - **revocation_assertion_responses**
```

Comment:
here the same as my previous comment, this parameter name must be changed

Reply:
As mentioned in the comment https://github.com/italia/eudi-wallet-it-docs/pull/308#discussion_r1658365625, the parameter name in changed.